Api/test/officeTest.js

```diff
@@ -0,0 +1,67 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+
+const expect = chai.expect;
+
+//Assertion
+chai.should();
+chai.use(chaiHttp);
+
+// To do
+describe('Office API', () => {
+  //Testing Different Routes 
+  // Get route
+  describe("Get /api/src/routes", () => {
+    it("It should Get all offices", (done) => {
+      chai.request(server)
+        .get("/api/src/routes")
```

Comment:
same here. even as a file path, it is incorrect since the api folder is has capital A. 

We shouldn't call `get` on what is not an endpoint.

Reply:
Done.